src/pages/Heeyun/Login/Login.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import './Login.scss';
+import { useState } from "react";
+import { useNavigate } from 'react-router'
+
+const Login = () => {
+    const [ID, setID] = useState('');
+    const [PW, setPW] = useState('');
+    const navigate = useNavigate(); 
+
+    function handleIdInput(event){
+        setID(event.target.value);
+        console.log(ID);
+    }
+
+    function handlePWInput(event){
+        setPW(event.target.value);
+        console.log(PW);
+    }
+
+    function goList(){
+        navigate('/list')
+    }
+
+    function changeButton(id,pw){
+        return id.includes('@') && pw.length>7 ? false : true
+    }
+    
+    const color = changeButton(ID,PW) ?  '#ADD3EA' : '#60ADED'
+
+    return (
+        <div className='Login' >
+            <div className='container'>
+                <section>
+                    <div>
+                        <h1>WeBucks</h1>
+                    </div>
+                    <form>
+                        <div>
+                            <input type="text" id="ID" className='boxes' placeholder="전화번호, 사용자 이름 또는 이메일" onChange = {handleIdInput} />
```

Comment:
가독성이 떨어지므로 단계를 잘 나눠주세요!
```suggestion
         <input 
              type="text" 
              id="ID" 
              className='boxes' 
              placeholder="전화번호, 사용자 이름 또는 이메일" 
              onChange = {handleIdInput} 
         />
```

Reply:
제가 봐도 알아보기 힘드네요 ㅋㅋㅋㅋ

Reply:
prettier 추천합니다 알아서 정리해줘요 ㅋㅋ